include/aspect/mesh_deformation/fastscape.h

```diff
@@ -0,0 +1,725 @@
+     */
+    void fastscape_init_f_(double *sf);
+
+    /**
+     * Set FastScape erosional parameters on land. These parameters will apply to the stream power law (SPL)
+     * and hillslope diffusion for basement and sediment. This can be set between timesteps.
+     */
+    void fastscape_set_erosional_parameters_(double *kkf,
+                                             const double *kkfsed,
+                                             const double *mm,
+                                             const double *nnn,
+                                             double *kkd,
+                                             const double *kkdsed,
+                                             const double *gg1,
+                                             const double *gg2,
+                                             const double *pp);
+
+    /**
+     * Set FastScape marine erosional parameters. This can be set between timesteps.
+     */
+    void fastscape_set_marine_parameters_(const double *sl,
+                                          const double *p1,
+                                          const double *p2,
+                                          const double *z1,
+                                          const double *z2,
+                                          const double *r,
+                                          const double *l,
+                                          const double *kds1,
+                                          const double *kds2);
+
+    /**
+     * Set advection velocities for FastScape. This can be set between timesteps.
+     */
+    void fastscape_set_v_(double *ux,
+                          double *uy);
+
+    /**
+     * Set FastScape uplift rate. This can be set between timesteps.
+     */
+    void fastscape_set_u_(double *up);
+
+    /**
+     * Set FastScape topography. This can be set between timesteps.
+     */
+    void fastscape_set_h_(double *hp);
+
+    /**
+    * Set FastScape basement. This can be set between timesteps. Sediment within FastScape
+    * is considered as the difference between the topography and basement, though this may differ
+    * from sediment as seen in ASPECT because the FastScape basement only takes the surface
+    * velocities into consideration.
+    */
+    void fastscape_set_basement_(double *b);
+
+    /**
+     * Get the current FastScape step iteration. This is set to zero when restarting FastScape.
+     */
+    void fastscape_get_step_(int *sstep);
+
+    /**
+     * Run FastScape for a single FastScape timestep.
+     */
+    void fastscape_execute_step_();
+
+    /**
+     * Create a .VTK file for the FastScape surface within the VTK folder of the
+     * ASPECT output folder.
+     */
+    void fastscape_named_vtk_(double *fp,
+                              const double *vexp,
+                              int *astep,
+                              const char *c,
+                              int *length);
```

Comment:
Strictly speaking, you can put anything you want here and the compiler will not complain -- that's because the compiler will simply take your word that that is the signature of the function, and it's up to *you* to make sure it matches what the Fortran function *really* does.